modules/gathering-data-specific-features.adoc

```diff
@@ -32,7 +32,7 @@ ifndef::openshift-origin[]
 |`registry.redhat.io/openshift-serverless-1/svls-must-gather-rhel8`
 |Data collection for OpenShift Serverless.
 
-|`registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel7`
+|`registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel8`
 |Data collection for Red Hat OpenShift Service Mesh.
 
 |`registry.redhat.io/rhcam-1-2/openshift-migration-must-gather-rhel8`
```

Comment:
Again, I can't comment on the specific line. For `prereqs` the second bullet says, `The OpenShift Container Platform (oc) installed.` To be consistent with the first bullet, consider phrasing in the imperative so both bullets are actions. So the rephrase would be, `Install the OpenShift Container Platform (oc).`